### PR TITLE
Update Octane and editions description

### PR DESCRIPTION
**Pull request by jenweber**
This is part of prep for Ember 4. Not much on this page needed to change.

### CODE REVIEW

File: app/templates/editions/index.hbs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <section class="container" aria-labelledby="ember-editions">
   <h1 id="ember-editions">Ember Editions</h1>
   <p>
+    An edition is a collection of features and patterns that represent
+    the best way to build Ember apps for a range of Ember versions.
```
Comment (ijlee2) [inline app/templates/editions/index.hbs:6]: I think the paragraph, as is, will become long to read. The paragraph includes the word `edition` too many times, which may lessen the impact. The following screenshot is at a desktop resolution.

Screenshot 2021-09-02 at 07 51 27

I think the sentence that you wrote is a key point and would like to emphasize it in its own paragraph.

```suggestion
    <em>An edition is a collection of features and patterns that represent
    the best way to build Ember apps for a range of Ember versions.</em>
  </p>
  <p>
```

Screenshot 2021-09-02 at 07 52 43

File: app/templates/editions/octane.hbs [hunk 2]
```diff
@@ -4,26 +4,21 @@
...
   <p class="intro">
-    It has arrived!
-    As of version 3.15 of Ember, when you generate a new app, you get everything you need to build your app using Octane.
-    You can read more about features and upgrade strategies in the
-    <a href="https://blog.emberjs.com/2019/12/20/octane-is-here.html">the official Ember blog post</a>.
+    Ember Octane is the latest edition in Ember.
+    An edition is a collection of features and patterns that represent
+    the best way to build Ember apps for a range of Ember versions.
+    Octane has modern, streamlined components and state management that make it fun
+    to build web applications.
```
Comment (ijlee2) [inline app/templates/editions/octane.hbs:11]:
```suggestion
    Octane has modern, streamlined components and state management so it's fun
    to build web applications.
```

File: app/templates/editions/octane.hbs [hunk 3]
```diff
@@ -4,26 +4,21 @@
...
+    JavaScript and HTML are at the front and center, enabling developers from many
```
Comment (ijlee2) [inline app/templates/editions/octane.hbs:12]: I didn't know the phrase `front and center` well till today. It's an adverb so I think it's better to use the phrase like this (or use another easier phrase for non-native speakers).

```suggestion
    JavaScript and HTML are front and center, enabling developers from many
```

File: app/templates/editions/octane.hbs [hunk 4]
```diff
@@ -4,26 +4,21 @@
...
+    different learning backgrounds to start building apps right away.
   </p>
+
+  <h2>Frequently Asked Questions</h2>
   <p class="mb-3">
-    Ember Octane describes a set of new features that, when taken together, represent a foundational improvement to the way you use Ember.
-    It has modern, streamlined components and state management that make it fun to build web applications.
+    When compared to older app styles, Ember Octane describes a set of new features that, when taken together, represent a foundational improvement to the way you use Ember.
```
Comment (ijlee2) [inline app/templates/editions/octane.hbs:18]: I think we can simplify this sentence like so:

```suggestion
    Ember Octane describes new features that, together, represent a foundational improvement to the way you use Ember.
```

File: app/templates/editions/octane.hbs [hunk 5]
```diff
@@ -44,54 +39,29 @@
...
-    </li>
-    <li>
-      <a href="https://github.com/ember-learn/super-rentals-tutorial">
-        The Tutorial
-      </a>
-    </li>
-    <li>
-      <a href="https://github.com/emberjs/ember.js">
-        Ember.js source code and API docs
+         Read the announcement
```
Comment (ijlee2) [inline app/templates/editions/octane.hbs:55]: Maybe clarify that the announcement is about Octane?

```suggestion
         Read the Octane announcement
```

File: app/templates/editions/octane.hbs [hunk 6]
```diff
@@ -100,11 +70,15 @@
   <dl class="dl-faq">
     <dt>Which release of Ember is Octane?</dt>
     <dd>
-      The Octane Edition was declared for Ember 3.15.
+      The Octane Edition was declared for [Ember 3.15 in December 2019](https://blog.emberjs.com/ember-3-15-released).
```
Comment (ijlee2) [inline app/templates/editions/octane.hbs:73]: `declared for` (currently present) sounded a bit awkward to me. Could we switch the order of the words?

```suggestion
      In December 2019, [Ember 3.15](https://blog.emberjs.com/ember-3-15-released) was declared to be Ember Octane.
```

File: app/templates/editions/octane.hbs [hunk 7]
```diff
@@ -100,11 +70,15 @@
...
     </dd>
     <dt>Do I need to refactor my whole app in order to use Octane features?</dt>
     <dd>
-      Good news - you do not need to change your whole app to use Octane's features! All are available for you to opt into, one piece at a time. Some features are even available in earlier versions.
+      Good news - you do not need to change your whole app to use Octane's features! 
+      All are available for you to opt into, one piece at a time. Some features are even available in earlier versions.
```
Comment (ijlee2) [inline app/templates/editions/octane.hbs:78]:
```suggestion
      You can opt into them, one at a time. Some features are even available in earlier versions.
```

File: app/templates/editions/octane.hbs [hunk 8]
```diff
@@ -121,6 +95,8 @@
       Yes! Octane is stable and well supported.
       It is the present and future of Ember, so you are saving yourself some upgrade work down the line.
       Be sure to budget some time for learning and experimentation.
+      From Ember 3.15 onwards, if you start a new app and use Ember CLI commands like `ember generate component` to create
+      your files, you are using Octane!
```
Comment (ijlee2) [inline app/templates/editions/octane.hbs:99]: I wasn't sure if this answer belongs well to the question `I'm starting a new app at work. Should I use Octane?`.

From this answer, it seemed to me like that there might be people who are unsure if their app is already Octane. _How could I check if I am using Octane?_

Did I misunderstand your intent?

Comment (jenweber) [reply app/templates/editions/octane.hbs:99]: I like the idea of making this be its own question. Thanks for the suggestion!

**Comment by jenweber** [comment]
@ijlee2 thank you for all the prose improvements! I still need to split that question out, and then this will be ready for another review/merging.

**Comment by jenweber** [comment]
I’ll have a look at this with fresh perspective to see what can be updated easily

**Pull request closed without merging**